Libraries/CommonFunctions.psm1

```diff
@@ -209,18 +219,38 @@ Function Add-SetupConfig {
         param ([System.Collections.ArrayList]$TestCollections, [string]$ConfigName, [string]$ConfigValue, [bool]$Force = $false)
         foreach ($test in $TestCollections) {
             if (!$test.SetupConfig.$ConfigName) {
-                # use CDATA when the value contains XML escaped characters
-                if ($ConfigValue -imatch "&|<|>|'|""") {
-                    $test.SetupConfig.InnerXml += "<$ConfigName><![CDATA['$ConfigValue']]></$ConfigName>"
-                } else {
-                    $test.SetupConfig.InnerXml += "<$ConfigName>$ConfigValue</$ConfigName>"
-                }
+                $test.SetupConfig.InnerXml += "<$ConfigName>$ConfigValue</$ConfigName>"
```

Comment:
did you sync latest code?

Reply:
thanks Lili.  I rebased base, but still don't know why it doesn't give me any warnings. after manual compare, updated. I also remember that previous PR has updated here for CDATA.